_includes/installing-vmware-webmks-containers.md

```diff
@@ -0,0 +1,55 @@
+### Installing VMware WebMKS on containers
+
+VMware vSphere Virtual Machine Remote console access requires the installation of the VMware WebMKS SDK.
+Due to license restrictions this SDK cannot be built in to {{ site.data.product.title_short }}
+
+To install the WebMKS SDK on containers we can take an existing container and create another layer on top of it.  Once built, this image will need to be pushed to a registry and the CR updated to reflect that.
+
+1. Download the latest WebMKS version (e.g. `WebMKS_SDK_{version}.zip`) from the VMware website.
+
+2. Set up a temporary directory where we will build the image
+
+   ```bash
+   $ mkdir -p /tmp/webmks_container/container-assets
+   ```
+
+3. Copy the tar file into the container-assets directory.  For example,
+
+   ```bash
+   $ cp ~/Downloads/WebMKS_SDK_*.zip /tmp/webmks_container/container-assets
+   ```
+
+4. Add the following Dockerfile as `/tmp/webmks_container/Dockerfile` and substitute the second `FROM` based on the image and version that you're running.
+
+   ```dockerfile
+   FROM registry.access.redhat.com/ubi8/ubi as webmks
+
+   COPY container-assets/ /webmks/
+
+   RUN cd /webmks && \
+       filename=$(find . -name WebMKS_SDK_*) && \
+       if [[ -f $filename ]] ; then unzip WebMKS_SDK_* ; fi
```

Comment:
Unzip isn't installed:
```
[1/2] STEP 3/3: RUN cd /webmks &&     filename=$(find . -name WebMKS_SDK_*.zip) &&     if [[ -f $filename ]] ; then unzip WebMKS_SDK_*.zip ; fi
/bin/sh: unzip: command not found
```

Probably easier to have the user unzip it to the /tmp directory